### PR TITLE
Create KNOWN_ISSUES.md (still working on it)

### PR DESCRIPTION
**Pull request by kings177**

### CODE REVIEW

**Comment by r614** [comment]
following this pr closely since IO usage/impl is missing in docs rn - happy to contribute once I mess around with this a bit more

File: KNOWN_ISSUES.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Known Issues
+
+## Can i run this on AMD/METAL GPUs?
+> TODO
+
+## Can i run this on windows?
+> TODO
+
+## How do i use IO?
+> TODO
```
Comment (developedby) [inline KNOWN_ISSUES.md:10]: In addition on how to do IO, how to do FFI?
> not as soon as IO, but planned or at least something similar to FFI.

File: KNOWN_ISSUES.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Known Issues
+
+## Can i run this on AMD/METAL GPUs?
+> TODO
```
Comment (developedby) [inline KNOWN_ISSUES.md:4]: "We'll support many other GPUs as soon as the CUDA version is sufficiently stable"

File: KNOWN_ISSUES.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+## Can i run this on windows?
+> TODO
```
Comment (developedby) [inline KNOWN_ISSUES.md:7]: For the moment, try using it through WSL2. We're working on it

File: KNOWN_ISSUES.md (hunk 1 above)
Comment (developedby) [inline KNOWN_ISSUES.md:10]: IO coming soon

**Review by developedby — COMMENTED** [review]
Libraries, packages, etc?
> We're adding a package manager soon

**Pull request merged**